src/main/java/org/generationcp/middleware/dao/dms/DmsProjectDao.java

```diff
@@ -786,8 +789,8 @@ public StudyMetadata getStudyMetadataForGeolocationId(final Integer geolocationI
 				studyMetadata.setTrialDbId(
 					(row[5] instanceof Integer) ? (Integer) row[5] : null);
 				studyMetadata.setTrialName((row[6] instanceof String) ? (String) row[6] : null);
-				studyMetadata.setStartDate((row[7] instanceof String) ? (String) row[7] : null);
-				studyMetadata.setEndDate((row[8] instanceof String) ? (String) row[8] : null);
+				studyMetadata.setStartDate(Util.tryParseDate((String) row[7]));
```

Comment:
Is it safe to use `Util.tryParseDate` if there is no date string returned from db?

Reply:
https://github.com/IntegratedBreedingPlatform/Middleware/pull/764#discussion_r392214326